### PR TITLE
Fix docker config for development

### PR DESCRIPTION
**Pull request by raisfathin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,12 +11,8 @@ RUN apt-get update -qq && \
 COPY binaries/cpdf/cpdf /usr/local/bin/
 RUN chmod +x /usr/local/bin/cpdf
 
-COPY trans/static/fonts/IRANSans/ttf/* /usr/local/share/fonts/
 COPY trans/static/fonts/SourceSansPro/ttf/* /usr/local/share/fonts/
-COPY trans/static/fonts/Korean/* /usr/local/share/fonts/
-COPY trans/static/fonts/Thai/* /usr/local/share/fonts/
-COPY trans/static/fonts/Taiwan/* /usr/local/share/fonts/
-COPY trans/static/fonts/Sinhala/* /usr/local/share/fonts/
+COPY trans/static/fonts/fontawesome/fonts/* /usr/local/share/fonts/
```
Comment (fushar) [inline Dockerfile:15]: Hmm, so in our ansible code we have something like this:

```
    - name: Manually install fonts bundled with app code
      copy:
        src: "{{ deploy_path }}/repo/trans/static/fonts/"
        dest: "/usr/share/fonts/"
        remote_src: true
```

Does this mean our ansible code is wrong, or can we make this Dockerfile to be similar to our ansible? (e.g. no need to specify individual fonts here)

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
```
Comment (fushar) [inline Dockerfile:1]: Can we use `focal` here? As we will use Ubuntu 20.04 in our deployment for actual IOI

(or `ubuntu:20.04` for more clarity)

**Review by fushar — CHANGES_REQUESTED** [review]
BTW, can we rebase to latest master? I've pointed it to `upstream/master`.

File: Dockerfile (hunk 1 above)
Comment (raisfathin) [reply Dockerfile]: Looks like both `/usr/local/share/fonts` and `/usr/share/fonts` work. Made it closer to the ansible code. Will send a followup PR to update the ansible code to use `/usr/local/share/fonts` as recent docs suggest to use that one.

File: Dockerfile (hunk 2 above)
Comment (raisfathin) [reply Dockerfile]: Done.

**Comment by raisfathin** [comment]
Rebased.

**Review by fushar — APPROVED** [review]

**Pull request merged**